server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/InternalRange.java

```diff
@@ -36,25 +37,31 @@
         protected final transient boolean keyed;
         protected final transient DocValueFormat format;
         protected final double from;
+        protected final Double originalFrom;
         protected final double to;
+        protected final Double originalTo;
         private final long docCount;
         private final InternalAggregations aggregations;
         private final String key;
 
         public Bucket(
             String key,
             double from,
+            Double originalFrom,
             double to,
+            Double originalTo,
```

Comment:
Could this one be always specified - a `double` instead of `Double`?

Reply:
Will see